src/components/LeftNav.js

```diff
@@ -140,10 +168,73 @@ class LeftNav extends Component<Props, State> {
 
   render() {
     return (
-      <Container>
-        <SearchContainer>
-          <SearchBar data={this.props.notes} />
+      <Container showFolders={this.props.showFolders}>
+        {this.props.showFolders && (
+          <SidebarContainer folder showFolders={this.props.showFolders}>
+            <View>
+              <Folder
+                folderName={'all notes'}
+                folderID={'all notes'}
+                isBeingEdited={false}
+                onDragOver={this.onDragOver}
+                onDrop={this.onDrop}
+                isOpen={this.props.activeFolder === 'all notes'}
```

Comment:
It makes me a little nervous having sentinel values like "all notes" in a condition, because it seems possible in a future refactor that we might change the name of the "all notes" folder to something else, then forget to change this condition. Is there another you could do this? Maybe an `isAllNotes(this.props.activeFolder)` function?

Reply:
Another thought: Maybe we need a data type that contains the possible types of these folders, i.e.
```
type Folder = "normal" | "empty" | "archive" | "all"
```

Then we could construct a note type like:
```
export type Note = {
  key: string,
  title?: string,
  content?: string,
  date?: number,
  folder: Folder,
  invisible?: boolean,
}
```